dataans/src/app_info/import.rs

```diff
@@ -0,0 +1,43 @@
+use leptos::*;
+
+use crate::backend::import::{import_app_data, select_file};
+
+#[component]
+pub fn Import() -> impl IntoView {
+    let toaster = leptoaster::expect_toaster();
+    let (is_importing, set_is_importing) = create_signal(false);
+
+    let import_data = move |_| {
+        if is_importing.get() {
+            return;
+        }
+
+        let toaster_clone = toaster.clone();
+        let set_is_importing_clone = set_is_importing.clone();
```

Comment:
As i remember, you don't need to clone it because Leptos signals implement the `Copy` trait. Can you try to compile without cloning?